test/command_callback/test_llc_command_callback.vader

```diff
@@ -0,0 +1,38 @@
+Before:
+  Save g:ale_llvm_llc_executable
+
+  unlet! g:ale_llvm_llc_executable
+  unlet! b:ale_llvm_llc_executable
+
+  runtime ale_linters/llvm/llc.vim
+
+  function! AssertHasPrefix(str, prefix) abort
```

Comment:
You'll need to delete this function in the `After` block, with `delfunction`. Otherwise it will be accessible from other tests.

Reply:
Fixed in 1e72204 :+1: